src/Microsoft.AspNet.Mvc.ApiExplorer/project.json

```diff
@@ -0,0 +1,19 @@
+﻿{
+  "description": "The API explorer component for ASP.NET MVC.",
```

Comment:
Maybe this? `Contains API explorer functionality for ASP.NET MVC for discovering metadata such as the list of controllers and actions, and their URLs and allowed HTTP methods.`